kubernetes/configuring-services.html.markerb

```diff
@@ -0,0 +1,80 @@
+---
+title: Configuring Services
+layout: docs
+toc: false
+nav: firecracker
+---
+
+<div class= "important icon">
+Fly Kubernetes is a closed beta. Email us at sales@fly.io for early access.
+</div>
+
+A [Kubernetes Service](https://kubernetes.io/docs/concepts/services-networking/service/) exposes applications running on your cluster.
+Fly Kubernetes supports ClusterIP and LoadBalancer Services.
+
+Using `kubectl`, we can create a Service. An example ClusterIP service
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: fksdemo-service
+spec:
+  selector:
+    app: fksdemo
+  ports:
+    - name: http
+      protocol: TCP
+      port: 80
+      targetPort: 8080
+```
+
+to create it, execute the command
+
+```
+kubectl apply -f service.yaml
+```
+
+## Fly.io connection Handlers
+
+Our platform supports a number of connection handlers. These are a sort of middleware that modify your connection before it reaches your application.
+More details on the available handlers can be found in our [documentation](https://fly.io/docs/networking/services/#connection-handlers). Connection
+handlers are supported via custom annotations to a Service object. The annotations have the form
```

Comment:
```suggestion
Fly.io connection handlers are processes that modify your connection before it reaches your application.
Learn more about [connection handlers](/docs/networking/services/#connection-handlers). Connection
handlers are supported with custom annotations to a Service object. The annotations have the form:
```

Reply:
processes or middleware? not sure which word is best

Reply:
hmm, i think we can actually just not give them a description. "Fly.io connections handlers modify your connection" reads well